cmd/vic-machine/create/create.go

```diff
@@ -196,8 +378,8 @@ func (c *Create) processParams() error {
 		return cli.NewExitError("key cert should be specified at the same time", 1)
 	}
 
-	if c.ExternalNetworkName == "" {
-		c.ExternalNetworkName = "VM Network"
+	if c.ClientNetworkName == "" {
```

Comment:
Let's have this in the argument definition as the default value rather than inline here.